src/Profiler/DataCollector/BlockDataCollector.php

```diff
@@ -16,45 +16,25 @@
 use Sonata\BlockBundle\Templating\Helper\BlockHelper;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
 
 /**
  * Block data collector for the symfony web profiling.
  *
  * @author Olivier Paradis <paradis.olivier@gmail.com>
  */
-class BlockDataCollector implements DataCollectorInterface, \Serializable
+class BlockDataCollector extends DataCollector
 {
     /**
      * @var BlockHelper
      */
     protected $blocksHelper;
 
-    /**
-     * @var array
-     */
-    protected $blocks = [];
```

Comment:
Removing protected/public stuff is a BC break :(

Reply:
I think this is ok since it's in a DataCollector. I know we should never assume what people do with a lib, but the worst that could happen is what happens at the moment with the issue ;) The profiler gives a error 500.